main.tf

```diff
@@ -27,7 +27,7 @@ resource "azurerm_key_vault_secret" "AZURE_APPINSIGHTS_KEY" {
 
 resource "azurerm_application_insights" "appinsights" {
   name                = "${var.product}-appinsights-${var.env}"
-  location            = var.appinsights_location
+  location            = "UK South"
```

Comment:
this should be `var.location` and the `appinsights_location` variable should be removed